Sources/Kevin/Core/KevinLocaleManager.swift

```diff
@@ -22,6 +22,7 @@ internal class KevinLocaleManager {
         "fi",
         "fr",
         "hu",
+        "is",
```

Comment:
I have added new locale to locale manage, however I was not sure whether it was needed to add support to Icelandic country in `KevinCountry`